.github/workflows/tag.yml

```diff
@@ -0,0 +1,43 @@
+name: Set git tag
+
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
+    inputs:
+      pushDateTag:
+        description: "Push datestamp (production) tag"
+        type: boolean
+        required: false
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  packages: write
+
+jobs:
+  set_tag:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: "main"
+          fetch-depth: 0
+
+      - name: Set git tag
+        run: |
+          tag=$(date +'%Y%m%d')
+          git config --global user.name 'GitHub Actions'
+          git config --global user.email 'github-actions@users.noreply.github.com'
+
+          if git tag -l $tag; then
+            echo "Tag $tag already exists."
+          else
```

Comment:
Do we need to do this? Can't we just let it override the tag if we ever hit that case?

Reply:
Yes you can't push an existing tag

Reply:
You're right, my bad